windows.Dockerfile

```diff
@@ -1,11 +1,21 @@
-FROM mcr.microsoft.com/windows/servercore:1809 as core
+FROM --platform=$BUILDPLATFORM golang:1.13.10-alpine3.10 as builder
+WORKDIR /go/src/github.com/Azure/secrets-store-csi-driver-provider-azure
+ADD . .
+ARG TARGETARCH
+ARG TARGETOS
+ARG LDFLAGS
+RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -ldflags "${LDFLAGS}" -o _output/secrets-store-csi-driver-provider-azure.exe ./cmd/
 
-FROM mcr.microsoft.com/windows/servercore:ltsc2019
+FROM mcr.microsoft.com/powershell:lts-nanoserver-1809
 LABEL description="Secrets Store CSI Driver Provider Azure"
-ARG IMAGE_VERSION=0.0.5
 
-COPY ./_output/secrets-store-csi-driver-provider-azure.exe /secrets-store-csi-driver-provider-azure.exe
-COPY ./install_windows.ps1 /install_windows.ps1
+COPY --from=builder /go/src/github.com/Azure/secrets-store-csi-driver-provider-azure/_output/secrets-store-csi-driver-provider-azure.exe /secrets-store-csi-driver-provider-azure.exe
 
 USER ContainerAdministrator
-ENTRYPOINT ["powershell", "C:\\install_windows.ps1"]
+CMD ["pwsh","-Command", "$global:TargetDir = $env:TARGET_DIR; \
```

Comment:
Do we have an e2e job that tests windows?

Reply:
Adding a pipeline for windows test. I'll open a PR tomorrow for adding it. We can merge these changes once the pipeline is merged.